### PR TITLE
possible issues

### PR DESCRIPTION
**Pull request by Jacfomg**
@Edoardo-Pedicillo possible issues on the flipping, can you have a look and confirm them ?

Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.
- [ ] Compatibility with Qibo modules (Please edit this section if the current pull request is not compatible with the following branches).
    - [ ] Qibo: `master`
    - [ ] Qibolab: `main`
    - [ ] Qibolab_platforms_qrc: `main`

### CODE REVIEW

**Comment by andrea-pasquale** [comment]
We have just made the release 😞

File: src/qibocal/protocols/characterization/flipping_signal.py [hunk 1]
```diff
@@ -178,7 +178,7 @@ def _fit(data: FlippingSignalData) -> FlippingSignalResults:
             popt[3] - x_min * 2 * np.pi * f / (x_max - x_min) * popt[2],
             popt[4] * 2 * np.pi * f / (x_max - x_min),
         ]
-        if popt[3] > np.pi / 2 and popt[3] < 3 * np.pi / 2:
+        if translated_popt[3] > np.pi / 2 and translated_popt[3] < 3 * np.pi / 2:
```
Comment (andrea-pasquale) [inline src/qibocal/protocols/characterization/flipping_signal.py:181]: I agree with all other changes, I need to check this one, but it seems correct.

Comment (Jacfomg) [reply src/qibocal/protocols/characterization/flipping_signal.py:181]: I think this was related to ramsey giving the correction in the wrong direction https://github.com/qiboteam/qibolab_platforms_qrc/pull/119#issuecomment-1948309606

**Review by igres26 — APPROVED** [review]
Looks good to me. Let's confirm @andrea-pasquale's comments and we should be able to merge.

File: src/qibocal/protocols/characterization/flipping_signal.py (hunk 1 above)
Comment (andrea-pasquale) [reply src/qibocal/protocols/characterization/flipping_signal.py:181]: Uhm, but this is `flipping` it is not `ramsey`...

Comment (Jacfomg) [reply src/qibocal/protocols/characterization/flipping_signal.py:181]: Yes sorry,

I think this was related to ~~ramsey~~ flipping giving the correction in the wrong direction.

**Comment by andrea-pasquale** [comment]
I manage correct the formula for the flipping. I am not sure if I hardcoded something related to the fact that this is a cavity and not a 2D resonator. Here are some reports:
http://login.qrccluster.com:9000/SA81DOUrR4OklXCYpMzLYA==
http://login.qrccluster.com:9000/Nc5uSr19QPyz0deZ9MXW8w==
http://login.qrccluster.com:9000/m8vpt5igQJapzsMY0iTbEw==
http://login.qrccluster.com:9000/5jHDWaknTDSDZioFMXsTmQ==

**Comment by Jacfomg** [comment]
> I manage correct the formula for the flipping. I am not sure if I hardcoded something related to the fact that this is a cavity and not a 2D resonator. Here are some reports: http://login.qrccluster.com:9000/SA81DOUrR4OklXCYpMzLYA== http://login.qrccluster.com:9000/Nc5uSr19QPyz0deZ9MXW8w== http://login.qrccluster.com:9000/m8vpt5igQJapzsMY0iTbEw== http://login.qrccluster.com:9000/5jHDWaknTDSDZioFMXsTmQ==

I can tell that the fits are fine, but have you checked that the amplitude correction was in the right direction, if so, let's merge it.

**Comment by andrea-pasquale** [comment]
I believe I checked but in some of the reports the amplitude correction seems to be wrong. It might be because I was just running qq fit and not updating the report. Let me double check.

**Comment by andrea-pasquale** [comment]
> I believe I checked but in some of the reports the amplitude correction seems to be wrong. It might be because I was just running qq fit and not updating the report. Let me double check.

Yes, one of the report was outdated, everything else works. I'll go ahead and merge.
@Jacfomg make sure to pull main when testing #720.

**Review by andrea-pasquale — APPROVED** [review]

**Comment by Jacfomg** [comment]
> > I believe I checked but in some of the reports the amplitude correction seems to be wrong. It might be because I was just running qq fit and not updating the report. Let me double check.
> 
> Yes, one of the report was outdated, everything else works. I'll go ahead and merge. @Jacfomg make sure to pull main when testing #720.

I will, but they should not overlap as we only changed the fitting here and acquisition on the other one, feel free to merge #720 as well.

**Comment by andrea-pasquale** [comment]
Yes, you are right. I just wanted to make sure that we can reproduce the same results with and without unrolling.

**Pull request merged**